server/models/thingy.js

```diff
@@ -9,7 +9,6 @@ const type = thinky.type;
 const schema = {
   id: type.string(),
   name: type.string().min(1),
-  price: type.number().min(0.01),
```

Comment:
OK, I was thinking of leaving it for possible future changes, but you're right... let's remove it. Any changes down the road shouldn't be too hard.